maas/plugins/horizon_check.py

```diff
@@ -111,5 +111,8 @@ def main(args):
         parser.add_argument('ip',
                             type=ipaddr.IPv4Address,
                             help='horizon dashboard IP address')
-        args = parser.parse_args()
+        parser.add_argument('site_name', 
+                            type=str, 
+                            help='Horizon Site Name')
+	args = parser.parse_args()
```

Comment:
Need to correct the indentation here so it matches the statements above and below (need one more space).